sdk/servicebus/service-bus/package.json

```diff
@@ -57,6 +57,7 @@
     "dotenv": "^7.0.0",
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^4.2.0",
+    "eslint-detailed-reporter": "~0.8.0",
```

Comment:
Should we maybe use the `^` instead of the `~` ?

Reply:
I agree we should remain consistent here with the other dev dependencies and allow rolling forward on minor version for this as well, so we should use `~`.

Reply:
Updated this to use ^